pkg/manager/impl/validation/validation.go

```diff
@@ -196,6 +203,11 @@ func validateParameterMap(inputMap *core.ParameterMap, fieldName string) error {
 	return nil
 }
 
+func ValidateProjectUpdateRequestLabelsAlphanumeric(request admin.Project) error {
```

Comment:
can you call this in the ValidateProjectRegisterRequest (create) call too? :D

Reply:
Good call!